### PR TITLE
Tagging ui

### PR DESCRIPTION
**Pull request by hardiesoft**
Now that the API support for this has been merged, we can use it from the bulk tagging interface!

### CODE REVIEW

**Comment by hardiesoft** [comment]
Still some bugs to iron out

**Comment by hardiesoft** [comment]
Seems to be all working as intended now.

**Comment by mjs** [comment]
It's not a showstopper but is there a way to get rid of the whitespace underneath the scrubber? 

image

It's fine when there's lots of tracks but when there's one or 2 the blank area looks a bit funny.

File: src/views/TaggingView.vue [hunk 1]
```diff
@@ -320,133 +302,42 @@ export default Vue.extend({
...
-          this.tracks = tracks;
-          this.currentRecordingData = recording;
-          // Get the first track without human tags
-          this.currentTrackIndex = this.orderedTracks.findIndex(
-            track => !this.trackIsAlreadyTaggedByHuman(track)
-          );
-        }
+    async getRecording(biasToDeviceId?: DeviceId): Promise<boolean> {
+      let recordingResponse;
+      if (biasToDeviceId !== undefined) {
```
Comment (mjs) [inline src/views/TaggingView.vue]: The sense here looks wrong. Shouldn't this be `if (biasToDeviceId === undefined) {`? That would explain why when trying out the feature it seemed to jump around a lot when tagging tracks without using the next recording button :)

Also, the `if` isn't even needed as `needsTag`'s argument is optional anyway.

**Pull request merged**